overlord/configstate/configcore/corecfg.go

```diff
@@ -57,17 +57,28 @@ var supportedConfigurations = map[string]bool{
 	"core.experimental.layouts": true,
 }
 
-func validateExperimentalSettings(tr Conf) error {
-	layoutsEnabled, err := coreCfg(tr, "experimental.layouts")
+func validateBoolFlag(tr Conf, flag string) error {
+	value, err := coreCfg(tr, flag)
 	if err != nil {
 		return err
 	}
-	switch layoutsEnabled {
+	switch value {
 	case "", "true", "false":
-		return nil
+		// noop
 	default:
-		return fmt.Errorf("experimental.layouts can only be set to 'true' or 'false'")
+		return fmt.Errorf("%s can only be set to 'true' or 'false'", flag)
 	}
+	return nil
+}
+
+func validateExperimentalSettings(tr Conf) error {
+	if err := validateBoolFlag(tr, "experimental.layouts"); err != nil {
+		return err
+	}
+	if err := validateBoolFlag(tr, "experimental.parallel-installs"); err != nil {
```

Comment:
I'd suggest that we call this 'instances' because, parallel is harder to spell and we never talk about parallelism, we talk about instances (to our users).

Reply:
OTOH the feature is dubbed as parallel installation of snaps, hence the proposed name. However, I don't feel strongly about it, so suggestions are certainly welcome. @mvo5 @pedronis what's your take?